cypress/component/annot-explanation.cy.js

```diff
@@ -61,7 +61,7 @@ describe("explanation", () => {
         cy.get(".btn").click();
 
         // Assert
-        cy.get(".card-body").should("be.visible");
+        cy.get(".card-body").should("be.visible",  { timeout: 8000 });
```

Comment:
Why is this part of the PR?

Reply:
This was inconsistent across different ci runs. For example, It failed [here](https://github.com/neurobagel/annotation_tool/actions/runs/6189686027/job/16805322365) when I was resolving the conflict.
It always passed locally so I checked the logs and it seemed to be timing out sometimes.

Reply:
ok, that's good to know. I'd say: add a short comment above this line to say that and then good to go